tutorials/setting-up-lamp.md

```diff
@@ -180,6 +180,73 @@ the local machine, and removing the test database.
 
     sudo mysql_secure_installation
 
+### Optional: Use phpMyAdmin for database administration
+
+You can use phpMyAdmin to administer your database through a UI.
+
+#### Install phpMyAdmin
+
+##### Debian/Ubuntu
+
+    sudo apt-get install phpMyAdmin
+
+##### CentOS 6 and 7
+
+    sudo yum install phpMyAdmin
+
+#### Configure phpMyAdmin
+
+##### Debian/Ubuntu
+  + Select **apache2**.
+  + Select **yes** to use `dbconfig-common` for database setup.
+  + Enter the database administrator's password that you chose during  MySQL configuration.
+  + Enter a password for the phpMyAdmin application.
+
+##### CentOS 6 and 7
+By default, phpMyAdmin allows connections from only localhost. To access the
+database from a workstation, modify the Allow directive in the Apache configuration file for phpMyAdmin.
+
+1. In your text editor, open `phpMyAdmin.conf`.
+
+       sudo nano /etc/httpd/conf.d/phpMyAdmin.conf
+
+1. To allow hosts on your network to access phpMyAdmin, [modify the Allow directive](https://httpd.apache.org/docs/current/mod/mod_access_compat.html#allow) for Apache 2.2. 
+For example, add your workstation's IP address to access phpMyAdmin from that IP.
+
+        <IfModule !mod_authz_core.c>
```

Comment:
This "looks like HTML" meaning that when this page is published, it will be stripped out by our security precautions.

If you want to show this code to the user, then you need to:

1. Rename `tutorials/setting-up-lamp.md` to `tutorials/setting-up-lamp/index.md`
1. Put the HTML-like code into a file in `tutorials/setting-up-lamp/`
1. Add a link to that file in `tutorials/setting-up-lamp/index.md`

Reply:
This comment applies to all HTML-like code snippets in your PR

Reply:
Does using a < within ` work? 

For example, to access phpMyAdmin from your workstation, add the workstation's IP address to `<IfModule !mod_authz_core.c>`.
       
        # Apache 2.2